DER/DerivativesContracts/Swaps.rdf

```diff
@@ -269,7 +299,7 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-der-drc-swp;hasLeg"/>
-				<owl:onClass rdf:resource="&fibo-der-drc-swp;StrikeLeg"/>
+				<owl:onClass rdf:resource="&fibo-der-drc-swp;FixedLeg"/>
```

Comment:
I don't see what about this sort of Swap would prevent it from having 2 variable legs e.g. it may reference a reference rate such as Fed Rate. If that's not true the definition should be updated or a note added

Reply:
I haven't seen that case anywhere in the literature - the ISDA definitions suggest that there is either a fixed payment stream or strike amount on one side and the other leg is variable.  We discussed it in the FCT a couple of times.  The original restriction was too narrow, only allowing for a fixed payment amount (i.e., a strike price), where in fact it could be a fixed rate of return, but we can ask again.  Perhaps there are outlier cases we haven't considered.  If this turns out to be true, we can raise another issue to address it.